laravel/the-basics/databases.html.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Databases
+layout: framework_docs
+objective: Notes and configurations on connecting with Databases
+order: 4
+---
+No application is complete without a data store!
+
+In this guide, you&#39;ll find configuration on different database applications and different approaches to connect them with your Laravel application in Fly.io.
```

Comment:
Do a quick find and replace to replace `&#39;` with an apostrophe `'` (Slab exports like this for some reason!)